ui/tests/acceptance/auth-test.js

```diff
@@ -123,6 +125,138 @@ module('Acceptance | auth', function (hooks) {
     }
   });
 
+  module('it sends the right payload when authenticating', function (hooks) {
```

Comment:
This module is basically a more robust version of `it sends the right attributes when authenticating` -- thoughts about replacing that module with this one?